web/models/notes.ex

```diff
@@ -0,0 +1,10 @@
+defmodule Notox.Note do
+  use Notox.Web, :model
+
+  schema "notes" do
+    field :title, :string
```

Comment:
This defaults to `:string`, so you could do `field :title`. Up to you though (some people prefer to be explicit).